### PR TITLE
Added to Documentation Markdown

### PR DESCRIPTION
**Pull request by jzf2101**
See #73

### CODE REVIEW

File: density/Documentation.md [hunk 1]
```diff
@@ -1,13 +1,20 @@
+##Overview
+Columbia has shared with ADI a live stream of the nubmer of devices connected to routers at various locations on campus. New counts are added every fifteen minutes. 
+
 ###Get an API Key
-API keys are available for Columbia univserity affiliates with valid email addresses `uni@*.columbia.edu` `uni@barnard.edu`
+API keys are [available](density.adicu.com/auth) for Columbia univserity affiliates with valid email addresses `uni@*.columbia.edu` `uni@barnard.edu`
```
Comment (natebrennand) [inline density/Documentation.md]: the link for API keys must include `http://...`, otherwise github tries to make it a link to something within the repo

**Comment by maclyn** [comment]
:+1:

File: density/Documentation.md [hunk 2]
```diff
@@ -1,27 +1,28 @@
+##Overview
+Columbia has shared with ADI a live stream of the nubmer of devices connected to routers at various locations on campus. New counts are added every fifteen minutes. 
+
 ###Get an API Key
-API keys are available for Columbia univserity affiliates with valid email addresses `uni@*.columbia.edu` `uni@barnard.edu`
+API keys are [available](http://density.adicu.com/auth) for Columbia univserity affiliates with valid email addresses `uni@*.columbia.edu` `uni@barnard.edu`
+
+Visit [density.adicu.com/auth](density.adicu.com/auth) and click on `get access`
```
Comment (natebrennand) [inline density/Documentation.md]: should have a period at the end

File: density/Documentation.md [hunk 3]
```diff
@@ -1,27 +1,28 @@
...
 
 
 ###Definitions
+Please see [http://desnity.adicu.com/docs/building_info](http://desnity.adicu.com/docs/building_info) for a table of the available building names, group names, building id's and group id's.
```
Comment (natebrennand) [inline density/Documentation.md]: s/desnity/density

**Comment by maclyn** [comment]
Number is misspelled on line 4.

**Comment by maclyn** [comment]
Spelling errors are fixed (I think). :+1:

**Comment by jzf2101** [comment]
I realize there's a font color issue:  @maclyn is working on it screen shot 2015-01-30 at 16 59 46

**Pull request merged**